### DISCUSSION TITLE
Label misalignment

### DISCUSSION THREAD

**Discussion by mariesvmadrazo** [formkit/formkit#1207] [General]
The FormKit of the project I'm working on right now is in version 1.5.3. The FormKit was working properly before but right now the labels are misaligned (see photo for reference)
![image](https://github.com/formkit/formkit/assets/48974172/57e96ade-8f75-4288-b001-118d4ace5b75)

Here is my formkit.config.js
![image](https://github.com/formkit/formkit/assets/48974172/6dc30a34-c865-4338-b671-292c534f7b11)

I also added :floating-label="true"
![image](https://github.com/formkit/formkit/assets/48974172/13c4df55-9f27-4ea2-9878-784b1655ee75)

I wonder how should I fix this?

**Comment by fenilli** [comment]
What version of the addons your project is currently on?

**Reply by mariesvmadrazo** [reply]
Version 1.5.3
![image](https://github.com/formkit/formkit/assets/48974172/0b31c6ae-2ed4-4cba-b7d3-46b0f5c41883)

**Reply by fenilli** [reply]
Strange, it might be because there could have been changes to the structure of the plugin and as you're using the old `@formkit/themes/genesis` so it is not updated to fix those, can you try changing to use the new themes builder?

**Reply by mariesvmadrazo** [reply]
I tried updating formkit to 1.5.9 but the issue persists. I also edited the photo of the discussion (the coinciding of mm/dd/yyyy and label for type date).
![image](https://github.com/formkit/formkit/assets/48974172/84784f0f-cb92-4611-80b0-9b84f75a9c04)

**Reply by fenilli** [reply]
I mean updating your config to use the new theme https://themes.formkit.com/

**Reply by mariesvmadrazo** [reply]
Can you help me understand how should I do it? I tried researching but can't find ways.

I also tried looking in the link you sent (https://themes.formkit.com/) and I didn't find similarity with the issue (mm/dd/yyyy coinciding with label (Regularization Date))
